### PR TITLE
Add new webinar

### PR DESCRIPTION
**Pull request by andrewpmiller**
## What this PR does:

- Adds a landing page for the upcoming webinar
- Resolves gymnasium/tracker#71

### CODE REVIEW

File: _data/webinars/web0013-decade-of-responsive-web-design.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 event_ID: web0013-decade-of-responsive-web-design
-event_title: "Responsive Design <span class=\"nowrap\">Ten Years</span> <span class=\"nowrap\">Later</span>"
+event_title: "A Decade of Responsive <span class=\"nowrap\">Web Design</span>"
```
Comment (andrewpmiller) [inline _data/webinars/web0013-decade-of-responsive-web-design.yml:2]: Thanks for catching this one @jgagne ;-)

File: _includes/webinar-bio.html [hunk 2]
```diff
@@ -11,9 +11,9 @@
     {%- assign headshot = bio.headshot -%}
 {%- endif -%}
 <section class="event-bio">
-    <img alt="{{ bio.name }}" src="{{ site.url }}{{ site.baseurl }}/img/webinar-bio-headshots/{{ headshot }}">
+    <img alt="" src="{{ site.url }}{{ site.baseurl }}/img/webinar-bio-headshots/{{ headshot }}">
```
Comment (mbifulco) [inline _includes/webinar-bio.html:14]: Why get rid of the alt text here?

**Review by mbifulco — APPROVED** [review]
Everything looks good to me - one possibly inconsequential question ~above~

File: _includes/webinar-bio.html (hunk 2 above)
Comment (andrewpmiller) [reply _includes/webinar-bio.html:14]: I believe it was generating a duplicate alt... but I'm not sure.

Comment (andrewpmiller) [reply _includes/webinar-bio.html:14]: Actually — @jgagne... 🤷‍♂️ why do we want it empty? And what would our friend Vano say?

Comment (jgagne) [reply _includes/webinar-bio.html:14]: In this context, a nearby heading provides the same text acting as a caption and an empty alt text avoids "Ethan Marcotte" or any other speaker or host's name from being repeated when read. `File under minor annoyances.`


wave
(Screenshot courtesy of @andrewpmiller)

**Review by jgagne — APPROVED** [review]

**Comment by mbifulco** [comment]
> In this context, a nearby heading provides the same text acting as a caption and an empty alt text avoids "Ethan Marcotte" or any other speaker or host's name from being repeated. `File under minor annoyances.`


wave
(Screenshot courtesy of @andrewpmiller) 

Fascinating! Very cool.

**Pull request merged**